pkg/signatures/public_key_verifier.go

```diff
@@ -71,6 +71,8 @@ func newCosignPublicKeyVerifier(config *storage.CosignPublicKeyVerification) (*c
 // as well as the claim verification of the payload of the signature.
 func (c *cosignPublicKeyVerifier) VerifySignature(ctx context.Context,
 	image *storage.Image) (storage.ImageSignatureVerificationResult_Status, error) {
+	log.Debugf("Verifying signatures of image %q with the following public keys: %+v",
+		image.GetName().GetFullName(), c.parsedPublicKeys)
```

Comment:
Do parsed public keys have proper line breaks within them? Ideally, we would want to have every public key start on a separate string